### ISSUE TITLE
feedback ass2

### ISSUE THREAD

**Issue by gosiamigut** [MiriamRiefel/MachineLearning#2]
Total 11.5 points out of 15 = 77%

Written assignment: 3.5/5
+ see the solutions online

Programming assignment: 8/10
+ well done
+ implementation
  + you could use a build-in function to standardize your data (as suggested in the assignment)
  + GridSearchCv already does the 10-fold cv for you, so you don't have to split the data beforehand
  + good use of functions
+ analysis
  + if you report score on the training set it's obvious there is no test set.
  + would be nice if you made a table to compare all the numbers.
  + when reporting on cv-set the standard deviation is also important
  + no confusion matrix
  + you didn't analyze the plots that you made
  + what are the optimal values of parameters that you used?
